packages/flutter/lib/src/material/slider.dart

```diff
@@ -421,10 +457,9 @@ class _RenderSlider extends RenderBox implements SemanticsActionHandler {
 
   void _updateLabelPainter() {
     if (label != null) {
-      // TODO(abarth): Handle textScaleFactor. https://github.com/flutter/flutter/issues/5938
       _labelPainter
         ..text = new TextSpan(
-          style: _textTheme.body1.copyWith(fontSize: 10.0),
+          style: _textTheme.body1.copyWith(fontSize: 10.0 * _textScaleFactor),
           text: label,
         )
         ..textDirection = textDirection
```

Comment:
The _reactionController field should presumably now be final, since there's no way to change it.